tests/antispoof/tests.c

```diff
@@ -266,6 +266,23 @@ static void test_antispoof_generic(const unsigned char **pkts,
 	pg_antispoof_arp_del_all(antispoof); 
 	REPLAY(0);
 
+	/* add ARP_MAX */
+       for (int i=0;i<=100;i++){
+	       if (i!=100){
```

Comment:
i < 100
so you can try with more than 101 iteration above:
```
for (int i=0;i< 150;i++){
	       if (i < 100){
```
Also, shouldn't we use a define instead of 100 ?
by that I mean a define in antispoof.h not the test